handoff.md

```diff
@@ -1,31 +1,155 @@
+##### This file is meant to help clear things up for the frontend repo and provide some direction as you start on the project. Be sure to also look at the Trello board user stories as well to know where you can start your work - that’s the best way to begin understanding the codebase. We know how overwhelming this can be at first. If you get stressed or have any questions and the answer is not in here, know that you can reach out to any member of the previous cohort on slack for help. Good luck this month! 
+
+### Product Review
+Feel free to watch our most recent product review meeting where we highlight some of the most recent work we’ve done and the next steps to take. https://www.youtube.com/watch?v=K92fUDY-xMQ&ab_channel=LambdaSchool 
 
 
 ### Quick Notes
-- We are using ant design for some styling/components.  https://ant.design/docs/react/introduce
-- Full walkthrough of the data flow for the form feature we implemented (including bugs to be fixed) https://youtu.be/dRBYRQ5QpoI
-- There may be CSS located in the node_modules file because of certain libraries being used...  If you want to change one of these components' styling, you may have to find the CSS in the Node_modules that is associated with it then overwrite that CSS by coding it in the src code.  node_modules is in the .gitignore file so any changes you make there will be ignored when you push your code.  If you have more questions please please please reach out to one of us because this concept was a bit weird and annoying to implement at first.
-- Testing coverage is not that great currently, that can always be improved upon
-- Redux is set up but hardly being used, feel free to convert some components to be utilizing this
-- If you UI/UX guys are feelin SPICY, then check this out https://d3js.org/. Could be a cool way to display the data rather than the current map we have.  Just make sure to let the stakeholder know before you actually implement it if you choose to do so
+#### UI/UX:
+We are using ant design for some styling/components.  https://ant.design/docs/react/introduce, Look at the code examples and the API documentation at the bottom of the pages for the component docs
+There may be CSS located in the node_modules file because of certain libraries being used...  If you want to change one of these components' styling, you may have to find the CSS in the Node_modules that is associated with it, then overwrite that CSS by coding it in the src code.  node_modules is in the .gitignore file so any changes you make here will be ignored when you push your code.  If you have more questions please please please reach out to one of us because this concept was a bit weird and annoying to implement at first.
+A lot of the css has repeats in the index.css, make sure to delete those extra and unused selectors as you go.
+Just a reminder that importing a css or less files in a react component doesn’t make those styles specific to that component, className is your friend.
```

Comment:
And these could use dashes to render as bullet points
```suggestion
- We are using ant design for some styling/components.  https://ant.design/docs/react/introduce, Look at the code examples and the API documentation at the bottom of the pages for the component docs
- There may be CSS located in the node_modules file because of certain libraries being used...  If you want to change one of these components' styling, you may have to find the CSS in the Node_modules that is associated with it, then overwrite that CSS by coding it in the src code.  node_modules is in the .gitignore file so any changes you make here will be ignored when you push your code.  If you have more questions please please please reach out to one of us because this concept was a bit weird and annoying to implement at first.
- A lot of the css has repeats in the index.css, make sure to delete those extra and unused selectors as you go.
- Just a reminder that importing a css or less files in a react component doesn’t make those styles specific to that component, className is your friend.
```

Reply:
Thanks Ethan, I accidently missed those. Made these bullet points!